packages/main/README.rst

```diff
@@ -398,22 +399,52 @@ First steps to start developing:
      to include it in the Robot **conda.yaml**
    - another possibility for Robocorp internal development is to use Robocorp **devpi** instance, by ``poetry publish --ci``
      and point **conda.yaml** to use rpaframework version in devpi
+
 #. ``poetry run python -m robot <ROBOT_ARGS> <TARGET_ROBOT_FILE>``
 
    - common *ROBOT_ARGS* from Robocorp Robot template: ``--report NONE --outputdir output --logtitle "Task log"``
+
 #. ``poetry run python <TARGET_PYTHON_FILE>``
 #. ``invoke lint`` to make sure that code formatting is according to **rpaframework** repository guidelines. It is possible and likely
    that Github action will fail the if developer has not linted the code changes. Code formatting is based on `black`_ and `flake8`_
    and those are run with the ``invoke lint``.
-#. the library documentation can be created in the repository root (so called "meta" package level)
+#. the library documentation can be created in the repository root (so called "meta" package level). The documentation is 
+   built by the docgen tools using the locally installed version of the project, in order to see recent changes, you must
+   delete the ``RPA`` folder and associated ``distinfo`` folders in the metapackage's **.venv** directory. The doc gen
+   command will reinstall the local packages.
 
    - ``poetry update``
    - ``make docs``
-   - open ``docs/build/html/index.html`` with the browser to view the changes
+   - open ``docs/build/html/index.html`` with the browser to view the changes or execute ``make local`` and navigate to
+     ``localhost:8000`` to view docs as a live local webpage.
+
+   **NOTE:** Due to the way optional packages are referenced in the metapackage's ``pyproject.toml``, they will not install
```

Comment:
As a side note, the other way around is `poetry install`-ing with the **meta** venv activated the optional packages (like `aws`) you want available for dev given the most recent version you have locally under the repo's project root dir. (this avoids re-generating a different _poetry.lock_ file or any change in the _*.toml_ one)